### PR TITLE
fix broken tests (kafka temp mock + test schema)

### PR DESCRIPTION
**Pull request by AfanasievYuriy**

### CODE REVIEW

File: taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/kafka/consumer/MessageConsumer.java [hunk 1]
```diff
@@ -10,7 +10,8 @@
 public class MessageConsumer {
     @KafkaListener(topics = "${laundry.management.topic.name}",
             groupId = "LaundryManagementServiceEventListener",
-            containerFactory = "laundryManagementListenerContainerFactory")
+            containerFactory = "laundryManagementListenerContainerFactory",
+            autoStartup = "${kafka.activateConsumers}")
```
Comment (andrsuh) [inline taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/kafka/consumer/MessageConsumer.java:14]: I like it! Maybe we just should add degault value here

File: taskcoordinator/app/src/main/resources/application.properties [hunk 2]
```diff
@@ -6,10 +6,13 @@ spring.datasource.password=123
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=validate
 spring.datasource.initialization-mode=always
+
 spring.jpa.properties.hibernate.default_schema=${database.schema.name}
 spring.flyway.schemas=${database.schema.name}
```
Comment (andrsuh) [inline taskcoordinator/app/src/main/resources/application.properties:11]: what is the difference between this properties? I mean... I understand but why the first one is not ehat we need?

File: taskcoordinator/app/src/main/resources/application.properties [hunk 3]
```diff
@@ -6,10 +6,13 @@ spring.datasource.password=123
...
+spring.flyway.placeholders.schema=${database.schema.name}
 
 # kafka properties
+kafka.activateConsumers=true
```
Comment (andrsuh) [inline taskcoordinator/app/src/main/resources/application.properties:17]: Please add really useful and comprehensive comments for each peoperty. I want the meaning if each property to be absolutely clear for everyone

File: taskcoordinator/app/src/main/resources/db/migration/V1.0.0__init.sql [hunk 4]
```diff
@@ -1,28 +1,28 @@
-CREATE SCHEMA IF NOT EXISTS task_coordinator;
+CREATE SCHEMA IF NOT EXISTS ${schema};
```
Comment (andrsuh) [inline taskcoordinator/app/src/main/resources/db/migration/V1.0.0__init.sql:1]: That's just cool!

File: taskcoordinator/app/src/main/resources/db/migration/V1.0.0__init.sql [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
 
-CREATE TABLE task_coordinator.laundries_state
+CREATE TABLE laundries_state
```
Comment (andrsuh) [inline taskcoordinator/app/src/main/resources/db/migration/V1.0.0__init.sql:3]: I understood why we need two properies

File: taskcoordinator/app/src/test/resources/application.properties [hunk 6]
```diff
@@ -9,4 +9,10 @@ spring.datasource.initialization-mode=never
 
 spring.jpa.properties.hibernate.default_schema=${database.schema.name}
 spring.flyway.schemas=${database.schema.name}
+spring.flyway.placeholders.schema=${database.schema.name}
+
+# kafka properties
```
Comment (andrsuh) [inline taskcoordinator/app/src/test/resources/application.properties]: comments

**Review by shinehlina — APPROVED** [review]

**Pull request merged**